src/Generator.php

```diff
@@ -72,10 +72,7 @@ function __construct(array $classDefinitions, $outputDir, $templateDir, $linkTem
      */
     function run()
     {
-        $loader = new Twig_Loader_Filesystem($this->templateDir, [
```

Comment:
Hi I g etting the same issue, but instead of removing the array of settings, you should move it as second argument of 
```php
$twig = new Twig_Environment($loader, [
    'cache' => false,
    'debug' => true,
]);
```